deploy/docker/Dockerfile-cypress

```diff
@@ -0,0 +1,48 @@
+# don't forget to pass environmet variables to docker run like this:
+# podman run -e CYPRESS_BASE_URL=<kiali_url> -e CYPRESS_PASSWD=<passwd> docker-cypress:latest
+# more details here https://github.com/kiali/kiali/tree/master/frontend/cypress
+
+# using same baseimage of node as its defined in package.json
+FROM cypress/base:14.17.0
+
+# Run as non-root user for MP+/OpenShift
+USER node
+
+ENV KIALI_HOME=/opt/kiali \
+    PATH=$KIALI_HOME:$PATH
+
+WORKDIR $KIALI_HOME
+
+# "Fake" dbus address to prevent errors. (is this still needed?)
+# https://github.com/SeleniumHQ/docker-selenium/issues/87
+ENV DBUS_SESSION_BUS_ADDRESS=/dev/null
+
+# A few environment variables to make NPM installs easier.
+# Good colors for most applications.
+ENV TERM xterm
+# Avoid million NPM install messages.
+ENV npm_config_loglevel warn
+# Allow installing when the main user is root.
+ENV npm_config_unsafe_perm true
+
+# copy frontend folder to container
+COPY --chown=node:node frontend/ $KIALI_HOME/
+# Install Cypress dependencies.
+RUN /usr/local/bin/yarn install --frozen-lockfile
+
+# Run Cypress tests.
+CMD ["yarn", "cypress:run:junit"]
+
+
+# TODO:
+# add oc and kubectl client binaries
+# log in clients above - all test using kubectl failed on `/bin/bash: kubectl: command not found`
+#
+# TODO:
+# shared folder with host for 
+# 1. test results (XML)
+# 2. videos and screenshots
+# 
+# TODO:
+# makefile target for build?
+# makefile target for run?
```

Comment:
@FilipB @jmazzitelli any suggestions please?

Reply:
base image is [Debian Buster](https://github.com/cypress-io/cypress-docker-images/blob/master/base/14.17.0/Dockerfile)